docs/user/arena-setup.rst

```diff
@@ -2,269 +2,272 @@
-been set up. Warps are set using the Warps tool.
-
-**Spawnpoints:** The *spawnpoints* of an arena are the locations where
-monsters can spawn from. MobArena will only spawn monsters from
-spawnpoints that have players nearby (in a 15-block radius). Note that
-the number of mobs spawned doesn't depend on how many spawnpoints an
-arena has - the number of mobs is determined by a formula (see
-[[Formulas]]) that only involves the wave number and player count
-(unless you use fixed amounts). Spawnpoints are added using the
-Spawnpoints tool.
-
-**Containers:** The *containers* of an arena are locations of chests,
-dispensers, etc. which contain items that will be renewed when the arena
-ends. Only registered containers will have their contents renewed, so it
-is not enough to simply put a chest in the arena - it also needs to be
-registered using the Chests tool.
+*****************
 
-Setup Mode
+There are four key parts to an arena:
+
+#. Lobby
+#. Arena floor
+#. Spectator area
+#. Exit point
+
+Lobby
+=====
+
+A lobby is where players choose a class before joining an arena. It's also the
+"waiting area" before a new match begins.
+
+Lobbies have two requirements:
+
+- **Class selection signs**: Clicked to select a class
+- **Iron block**: Clicked to mark self as "ready"
+
+Class selection signs must have the name of the class on the first line
+(case-sensitive). The last three lines are not checked and they can have any
+text. You will not receive a confirmation message after making a new sign since
+MobArena does not register the creation of class selection signs.
+
+Arena floor
+===========
+
+An arena floor is where the action happens. Players fight through mob waves on
+the arena floor. How the arena floor looks is up to you, but we recommend these
+minimum requirements:
+
+- Closed in by walls on all sides
+- Have a ceiling / roof
+
+This prevents players and mobs from escaping and also blocks players from
+wandering out of the arena with class items.
+ander out of the arena.
+
+Spectator area
+==============
+
+A spectator area lets non-players watch an on-going match. The ``/ma spec``
+command teleports a player into the spectator area. If configured, [#]_ players
+warp into the spectator arena when they die.
+
+Design the area so spectators cannot escape the spectator area, since they
+invincible. Spectators should *not* enter the arena floor or exit the spectator
+area on foot.
+
+.. [#] Set ``spectate-after-death`` to ``true`` in the config file to force
+   players to the spectator area after dying
+
+Exit point
+==========
+
+When players leave the arena or when the last player standing dies, arena
+players and spectators teleport to the location they joined from. Optionally, an
+arena can have an exit warp. This controls where players go after leaving a
+match.
+
+
+****************
+Defining regions
+****************
+
+When an arena is ready, it must be **defined** into…
```

Comment:
Not a big fan of this wording. The "readiness" of an arena in this context comes down to whether or not it has been built. So maybe "When an arena has been built, [...]"? I also think the rest of the line is a bit too rushed. Maybe allude to the fact that this part of the process is now "MobArena territory", so to speak. The building of an arena has nothing to do with MobArena, but the regions, warps, etc. do. Perhaps something along the lines of "[...] it must be set up in MobArena by defining..."